Automattic/newspack-plugin
# Add the Significant Revisions feature

### All Submissions:






### Changes proposed in this Pull Request:



Closes #1732 . (But not covering the "additionally the date/time for the last significant revision would be used as the "last updated date" on the post " request)

### How to test the changes in this Pull Request:

See the test instructions on https://github.com/Automattic/newspack-plugin/pull/2030 for set up and base test instructions

* In the Revisions screens, Mark some revisions as significant
* Run the tests as the instructions above and make sure the significant revisions are never deleted
* Open the "Help" section on the top of the page and see the "Nespack" section
* read it and confirm it's clear and that it has no typos
* get the ID of one of the revisions marked as significant from the address bar
* Delete this revision directly from the database `DELETE FROM wp_posts WHERE ID = %d`
* Go back to the revisions page and confirm you see a Warning telling you about the deleted revision


![Captura de tela de 2022-09-23 17-18-41](https://user-images.githubusercontent.com/971483/193136802-930c755c-76a2-42d4-9448-1920a2ecf2f6.png)

* Click to restore the backup and make sure the revision is restored


### Other information:
